public/js/home.js

```diff
@@ -1264,6 +1264,31 @@ document.getElementById("default_open").click();
 
 //#ifndef lite
 $.ajax("/version").then(ver => $("#version").text(ver));
+
+$.ajax("/updates").then(upt => {
+	$("#updates").html(upt);
+	document.getElementById("changelog").outerHTML = "<h2 class='info-header'>Version History/What's New:</h2>";
+    var items = document.getElementById("updates").getElementsByTagName("h2");
```

Comment:
Use `const` instead of `var` and [`document.querySelectorAll`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll) instead of `getElementById` and `getElementsByTagName`:
```suggestion
    const items = document.querySelectorAll("#updates h2");
```